### PR TITLE
yggdrasil: bump to 0.3.6

### PR DESCRIPTION
**Pull request by wfleurant**
Signed-off-by: William Fleurant <meshnet@protonmail.com>

Maintainer: @wfleurant
Compile tested: x86/64, snapshots
Run tested: x86/64, snapshots

Description: bump version to 0.3.6

### CODE REVIEW

**Comment by neheb** [comment]
Just force push if it fails.

**Comment by wfleurant** [comment]
okay should be all set and includes @BKPepe fixups. thanks.

File: net/yggdrasil/Makefile [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 PKG_SOURCE_URL:=https://codeload.github.com/yggdrasil-network/yggdrasil-go/tar.gz/v$(PKG_VERSION)?
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_HASH:=2c69029adeb053ad049e90f1e4b7efa986094779868da77464d3c869984e861b
+PKG_HASH:=dc1699064319f19a64ac57bac366a15d718008fdb75ef03bf4252d3552dff4eb
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-go-$(PKG_VERSION)
 
-PKG_LICENSE:=GPL-3.0
 PKG_MAINTAINER:=William Fleurant <meshnet@protonmail.com>
+PKG_LICENSE:=GPL-3.0-or-later
```
Comment (neheb) [inline net/yggdrasil/Makefile]: Add PKG_LICENSE_FILES:=LICENSE below

File: net/yggdrasil/Makefile [hunk 2]
```diff
@@ -1,16 +1,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=yggdrasil
-PKG_VERSION:=0.3.5
-PKG_RELEASE:=4
+PKG_VERSION:=0.3.6
+PKG_RELEASE:=1
 
 PKG_SOURCE_URL:=https://codeload.github.com/yggdrasil-network/yggdrasil-go/tar.gz/v$(PKG_VERSION)?
```
Comment (neheb) [inline net/yggdrasil/Makefile]: Swap the order of PKG_SOURCE and PKG_SOURCE_URL

**Comment by neheb** [comment]
Also, what is up with the init script. It has to call an instance of /bin/ash ???

File: net/yggdrasil/Makefile (hunk 1 above)
Comment (wfleurant) [reply net/yggdrasil/Makefile]: Thanks neheb,  

> - [ ] PKG_LICENSE new value `LGPL-3.0-only`.
[yggdrasil-go/LICENSE](https://github.com/yggdrasil-network/yggdrasil-go/blob/master/LICENSE)
```
This software is licensed under the LGPLv3, included below.
.... 8< .... 8< .... 8< .... 8< .... 8< .... 8< .... 8< .... 8< .... 
                   GNU LESSER GENERAL PUBLIC LICENSE
                       Version 3, 29 June 2007
.... 8< .... 8< .... 8< .... 8< .... 8< .... 8< .... 8< .... 8< .... 
  Each version is given a distinguishing version number. If the
Library as you received it specifies that a certain numbered version
of the GNU Lesser General Public License "or any later version"
applies to it, you have the option of following the terms and
conditions either of that published version or of any later version
published by the Free Software Foundation. 
```

File: net/yggdrasil/Makefile (hunk 2 above)
Comment (wfleurant) [reply net/yggdrasil/Makefile]: its now matching same order as contributing.md

**Comment by wfleurant** [comment]
@neheb > Also, what is up with the init script. It has to call an instance of /bin/ash ???

The init script(s) are from openwrt-routing/cjdns where this addressed a procd ~ cjdroute bug.

**Comment by yousong** [comment]
Thank you

**Pull request merged**